docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| ctype            |     |     |     | *   | *   | *   | *   | *   | *   |
+| curl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| dba              |     |     |     | *   | *   | *   | *   | *   | *   |
+| dom              |     |     |     | *   | *   | *   | *   | *   | *   |
+| enchant          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| event            |     |     |     |     | *   | *   | *   | *   |     |
+| exif             |     |     |     | *   | *   | *   | *   | *   | *   |
+| ffi              |     |     |     |     |     |     |     | *   | *   |
+| fileinfo         |     |     |     | *   | *   | *   | *   | *   | *   |
+| ftp              |     |     |     | *   | *   | *   | *   | *   | *   |
+| gd               | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   | *   |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   | *   |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| imap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| interbase        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| intl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
+| msgpack          |     |     | *   | *   | *   | *   |     | *   |     |
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     |     |
+| mysqli           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mysqlnd          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| newrelic         |     |     | *   | *   | *   | *   | *   | *   | *   |
+| oauth            |     |     |     | *   | *   | *   | *   | *   |     |
```

Comment:
Pourquoi?

Reply:
Haha it merged but not released yet it should be bientôt :pray:

Reply:
I've updated the image and the doc will be included on the next php8 image release